aten/src/ATen/native/group_norm.cpp

```diff
@@ -237,8 +236,7 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> math_group_norm(
       /*training=*/true,
       /*momentum=*/0,
       eps);
-  at::Tensor out = std::get<0>(outputs);
-  out = out.view(input_shape);
+  at::Tensor out = std::get<0>(outputs).view(input_shape);
```

Comment:
Why not use auto here?
```suggestion
  auto out = std::get<0>(outputs).view(input_shape);
```